src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4117,4 +4093,75 @@ export class StudyViewPageStore {
         default: []
     });
 
+    readonly mutationAsClinicalData = remoteData<ClinicalData[]>({
+        await: () => [this.samples],
+        invoke: async () => {
+            const filteredSamples = await internalClient.fetchFilteredSamplesUsingPOST({
+                studyViewFilter: Object.assign({}, this.initialFilters, { withMutationData: true })
+            });
+
+            const filteredSampleByKey = _.keyBy(filteredSamples, sample => sample.uniqueSampleKey);
+
+            return Promise.resolve(this.samples.result.map(sample => {
+                return {
+                    clinicalAttributeId: UniqueKey.WITH_MUTATION_DATA,
+                    patientId: sample.patientId,
+                    sampleId: sample.sampleId,
+                    studyId: sample.studyId,
+                    uniquePatientKey: sample.uniquePatientKey,
+                    uniqueSampleKey: sample.uniqueSampleKey,
+                    value: filteredSampleByKey[sample.uniqueSampleKey] !== undefined ? Datalabel.YES : Datalabel.NO
+                } as ClinicalData
+            }));
+        },
+        onError: (error => { }),
```

Comment:
these onErrors will cause them to fail silently so we need to make sure error condition is handled in code